apps/ai-pipeline/app/pipeline/node_generator.py

```
@@ -0,0 +1,234 @@
+            if attempt == settings.retry_count:
+                # 폴백: 기본 루트 노드
+                logger.warning("Root 생성 실패, 폴백 사용: %s", str(e)[:100])
+                return GenerationResult(
+                    node_type="narrative",
+                    narrative_text=f"당신의 휴대폰에 알 수 없는 번호로 연락이 왔습니다. {phishing_type} 관련 의심스러운 내용입니다.",
+                    choices=[
+                        ChoiceResult(text="응답한다", is_dangerous=True, resource_effect={"trust": 1, "money": 0, "awareness": 0}),
+                        ChoiceResult(text="무시한다", is_dangerous=False, resource_effect={"trust": -1, "money": 0, "awareness": 1}),
+                    ],
+                    image_prompt="A middle-aged Korean person in casual home clothes, receiving a suspicious phone call, tense atmosphere, modern Korean apartment living room, dark lighting, webtoon style illustration, no text, no letters",
+                    reasoning=f"LLM 호출 실패로 폴백 노드 생성: {str(e)}"
```

Ruff (0.15.12)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)